teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>Add a markdown cell above:</p><p>'First, we combine the names and SMILES of the molecules, together with their structure, in a dataframe.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done.</p>